packages/accounts-passwordless/server_tests.js

```diff
@@ -4,6 +4,10 @@ import { SHA256 } from 'meteor/sha';
 
 const USER_TOKEN = '123ABC';
 
+// The test suite for accounts-passwordless includes testing whether it gets the right error messages from the server.
+// So, we need this disabled, otherwise those tests incorrectly fail when you run them.
+Accounts._options.ambiguousErrorMessages = false;
```

Comment:
Wait, but tests were already passing:

<img width="520" alt="image" src="https://github.com/user-attachments/assets/e4dc2467-0828-407e-b063-40d901c043cf">

Reply:
I tested them locally on the `release-3.1.0` branch and they are failing here without the changes. 🤔

### testing package `accounts-passwordless` on release-3.1.0 as of 19/11/2024
![tests-failing](https://github.com/user-attachments/assets/a3c61489-456e-4681-ae90-5fb78f01941c)


### testing package `accounts-passwordless` on release-3.1.0 + changes 
![tests-passing](https://github.com/user-attachments/assets/0ac4c48f-989b-4c08-9627-13ebed3cb5fd)

Maybe the configuration is different if you're testing multiple packages? I see tests for `base64` in your screenshot.